docs/user-guides/cluster-role-binding.md

```diff
@@ -0,0 +1,112 @@
+# Cluster Role Configuration
+
+The cluster role contains rules that represent a set of permission to grant access to a specific target depending on the binding rule.
+
+This page describes why we need a cluster role for the Vald cluster.
+
+## What are cluster role and cluster role binding for the Vald cluster?
+
+In Vald, the index is distributed across the cluster depending on the resource usage of the cluster, it requires settings to grant permission to a specific role to retrieve cluster information on Kubernetes.
+
+In Vald, the settings are deployed automatically when using helm to deploy.
+
+- [clusterrole.yaml](https://github.com/vdaas/vald/blob/main/k8s/discoverer/clusterrole.yaml)
+- [clusterrolebinding.yaml](https://github.com/vdaas/vald/blob/main/k8s/discoverer/clusterrolebinding.yaml)
+
+These configurations allow the service account `discoverer` to access different resources in the Kubernetes cluster.
+
+### Cluster role settings
+
+By looking at the [cluster role settings](https://github.com/vdaas/vald/blob/main/k8s/discoverer/clusterrole.yaml), the access right of the following resources are granted to service account `discoverer`.
+
+```yaml
+rules:
+  - apiGroups:
+      - apps
+    resources:
+      - replicasets
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+      - nodes/proxy
+      - services
+      - endpoints
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
+  - nonResourceURLs:
+      - /metrics
+    verbs:
+      - get
+  - apiGroups:
+      - "metrics.k8s.io"
+    resources:
+      - nodes
+      - pods
+    verbs:
+      - get
+      - list
+```
+
+All of these rules are required to retrieve Nodes and Pods resource usage from [kube-apiserver](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/) and also used to discover new Vald Agents or Nodes created on the cluster.
+
+### Cluster role binding settings
```

Comment:
What would you like to describe?
For cluster role binding only discoverer, or overall?
It should be formatted the section title and sentences.

Reply:
fixed by changing the title